### ISSUE TITLE
Support yapf ignore configuration in pyproject.toml

### ISSUE THREAD

**Issue by dclong** [google/yapf#904]
Now that yapf supports formatting configuration in pyproject.toml, it's appealing to support yapf ignore configuration in pyproject.toml too so that we can configure everything about yapf in pyproject.tom.

**Comment by dclong** [comment]
The PR https://github.com/google/yapf/pull/937 addresses this issue. However, it uses a triple quotes string to specify multiple patterns. It is more nature to allow users to specify a list of patterns. Anyway, the code tries to parse the triple quotes string to a list of patterns. 
In addition to the following configuration style, 
```
[tool.yapfignore]
ignore_patterns="""
temp/**/*.py
temp2/*.py
"""
```
allowing the below configuration style as well.
```
[tool.yapfignore]
ignore_patterns = [
    "temp/**/*.py",
    "temp2/*.py"
]
```

**Comment by gwelymernans** [comment]
PR #937 uses the `toml` module to do the parsing. Does that not have support for a list of patterns?

**Comment by hirosassa** [comment]
Thanks for your suggestion. I'll try it.

**Comment by hirosassa** [comment]
@dclong Fixed the issue in #946 

@gwelymernans thanks for merging the PR!
